Inference:
    ros__parameters:
        model_name: "1.onnx"
        fps: 50
        vx: 0.0
        vy: 0.0
        dyaw: 0.0
        cycle_time: 0.70
        obs_scales_lin_vel: 2.0
        obs_scales_ang_vel: 1.0
        obs_scales_dof_pos: 1.0
        obs_scales_dof_vel: 0.05
        motor_lower_limit: [-0.15,-1.0,-1.0,-0.4,-1.0,-0.5,-0.7,-1.2,-1.0,-1.5,-1.0,-0.5]
        motor_lower_limit: [0.7, 1.2, 1.0, 1.5, 1.0, 0.5, 0.15, 1.0, 1.0, 0.4, 1.0, 0.5]

